ios/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 1.1.0
+version: 1.1.1
 apiversion: 2
 architectures: arm64 x86_64
```

Comment:
```suggestion
 architectures: arm64 x86_64 i386 armv7
```

Build is erroring that i386, and armv7 are built into the vinary, but not included in the manifest

```
[2019-09-13T06:58:44.206Z] [ERROR] : There is discrepancy between the architectures specified in module manifest and compiled binary.
[2019-09-13T06:58:44.206Z] [ERROR] : Architectures in manifest: arm64, x86_64
[2019-09-13T06:58:44.206Z] [ERROR] : Compiled binary architectures: i386, x86_64, armv7, arm64
[2019-09-13T06:58:44.206Z] [ERROR] : Please update manifest to match module binary architectures.
```